lib/helium/client/configurations.rb

```diff
@@ -0,0 +1,31 @@
+module Helium
+  class Client
+    module Configurations
+
+      def configurations
+        Configuration.all(client: self)
+      end
+
+      def configuration(id)
+        Configuration.find(id, client: self)
+      end
+
+      def config_device_configurations(id)
+        path = "/configuration/#{id}/device-configuration"
+        response = get(path)
+        device_confs_data = JSON.parse(response.body)["data"]
+
+        device_confs = device_confs_data.map do |dc|
+          DeviceConfiguration.new(client: self, params: dc)
+        end
+
+        return device_confs
+      end
+
+      # Configurations are immutable, so no updates are available
```

Comment:
maybe resource should have an immutable true/false property that prevents attempting patch calls when it's set to true? can leave that for later refactoring though